pywt/_multilevel.py

```diff
@@ -825,3 +828,368 @@ def array_to_coeffs(arr, coeff_slices, output_format='wavedecn'):
                 "Unrecognized output format: {}".format(output_format))
         coeffs.append(d)
     return coeffs
+
+
+def _check_fswt_axes(data, axes):
+    """Axes checks common to fswt, ifswt."""
+    if len(axes) != len(set(axes)):
+        raise ValueError("The axes passed to fswt must be unique.")
+    try:
+        [data.shape[ax] for ax in axes]
+    except IndexError:
+        raise ValueError("Axis greater than data dimensions")
+
+
+class TransformResult(object):
```

Comment:
This name is probably too generic if the class is specific to FSWT. How about something like `FswtResult`?